__tests__/mint-tokens.test.js

```diff
@@ -0,0 +1,66 @@
+import TestUtils from "./test-utils";
+
+describe("mint-tokens api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/mint-tokens")
+      .send({
+        token: "03",
+        amount: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
+  });
+
+  it("should create a token with amount as string", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/mint-tokens")
+      .send({
+        token: "03",
+        amount: "1",
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
+  });
+
```

Comment:
I feel we should test minting 0.01 tokens, which will require a 0.01 HTR deposit.

Reply:
I am already testing minting 0.01 tokens (amount: 1), but as the `fullnode` response is mocked, I am unable to assert the 0.01 HTR deposit, unless I mock a response exactly with 0.01 HTR deposit, but it seems to be an invalid effort.